packages/firebase_core/lib/firebase_app_desktop.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Invertase Limited. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file.
+
+part of firebase_core_desktop;
```

Comment:
Maybe
```suggestion
part of firebase_core_tizen;
```

You might also rename this file to `firebase_app_tizen.dart`.